lib/form_data.js

```diff
@@ -354,6 +355,9 @@ FormData.prototype.submit = function(params, cb) {
     , defaults = {method: 'post'}
     ;
 
+  var proxy = process.env.HTTPS_PROXY;
```

Comment:
Shouldn't both proxy types HTTP and HTTPS be handled depending on options.protocol value ?